readme.md

```diff
@@ -1,3 +1,51 @@
-## Read Me
+# RESTAURANT PAGE
 
-## On Development Branch
+A simple restaurant page built using Vanilla Javascript to generate its entire content.
+
+![screenshot](screenshot.png)
+
+
+## Built With
+- JavaScript
+- Webpack
+- HTML & CSS
+
+## Getting Started
+To get a copy up and running follow these simple example steps:-
+
+- Clone https://github.com/Alaska01/Restaurant_Page.git to your local machine.
+- cd into the restaurant-page directory and navigate into the dist directory.
+
+
+## Usage
+- Clone the project to your local machine
+- Run npm install
+- npm run watch
+
+
+### Live Demo
+<a href="https://aye-restaurant-page.netlify.app/">Click here</div>
+
+
+## Author
+- Github: https://github.com/Alaska01
+- LinkedIn: https://www.linkedin.com/in/daniel-asoo-aye/
+- Twitter: https://twitter.com/AyeAsoo
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Alaska01/Restaurant_Page/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Google
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
```

Comment:
- [ ] Kindly fix the broken MIT link.
Please note that such small mistakes might leave bad impressions on future viewers. 🙏 😄